spec/helpers/registered_applications_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
Only generate these files if you need them... I can tell you are using a generator. I would only use a generator for creating migrations. Not models. You don't want all of these extra files. Just the ones that you want.

Reply:
So don't do `rails g model ModelName` ?